.github/workflows/build-linux-arm64-installer.yml

```
@@ -44,6 +44,16 @@ jobs:
         echo "::set-output name=CHIA_INSTALLER_VERSION::$(python3 ./build_scripts/installer-version.py)"
         deactivate
 
+    - name: Test for secrets access
+      id: check_secrets
+      shell: bash
+      run: |
+        unset HAS_SECRET
+        if [ -n "$SECRET" ]; then HAS_SECRET='true' ; fi
+        echo ::set-output name=HAS_SECRET::${HAS_SECRET}
+      env:
+        SECRET: "${{ secrets.test_pypi_password }
```

```suggestion
        SECRET: "${{ secrets.TEST_PYPI_PASSWORD }
```


Pretty sure theyse are all caps